openmrs-19/src/test/java/org/motechproject/openmrs19/it/MRSEncounterServiceIT.java

```diff
@@ -116,6 +116,13 @@ public void shouldCreateEncounter() throws HttpException, ParseException, Interr
         assertEquals(encounter.getLocation().getUuid(), mrsListener.eventParameters.get(EventKeys.LOCATION_ID));
         assertEquals(encounter.getEncounterDatetime(), mrsListener.eventParameters.get(EventKeys.ENCOUNTER_DATE));
         assertEquals(encounter.getEncounterType().getUuid(), mrsListener.eventParameters.get(EventKeys.ENCOUNTER_TYPE));
+
+        Encounter fetchedEncounter = encounterAdapter.getEncounterByUuid(DEFAULT_CONFIG_NAME, encounter.getUuid());
+        Observation fetchedObservation = obsAdapter.getObservationByUuid(DEFAULT_CONFIG_NAME, fetchedEncounter.getObs().get(0).getUuid());
```

Comment:
Probably it is necessary to fetch this observation, however please try compare observation values from created encounter. Can you write here which fields are exposed in created json from OpenMRS server